test/integration/api/forms/forms.js

```diff
@@ -295,7 +299,35 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
             .expect(200)
             .then(({ body }) => {
               body.publishedAt.should.be.a.recentIsoDate();
-            })))));
+              body.publishedAt.should.eql(body.createdAt);
+              (body.updatedAt == null).should.equal(true);
```

Comment:
I just wanted to check that `updatedAt` would be null when the form is published in the same request as it is created. The test below is the separate case. 

There might be other tests that already exist that are very similar.